parking_permits/templates/emails/vehicle_low_emission_discount_deactivated.html

```diff
@@ -1,14 +1,31 @@
-{% extends "emails/base.html" %}
 {% load i18n %}
+<!DOCTYPE html>
+<html lang="fi">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <title></title>
+    </head>
 
-{% block title %}{% translate "Vehicle discount right expired" %}{% endblock %}
-
-{% block content %}
-    <p>
-        {% translate "The right to a discount has been removed from the vehicle." %}
-    </p>
-    <p>
-        {% translate "Vehicle" %}: {{ permit.vehicle.registration_number }}<br>
-        {% translate "Discount right ends" %}: {% now "j.n.Y, H:i" %}
-    </p>
-{% endblock %}
+    <body>
+        <div>
+            <div>{% translate "The right to a discount has been removed from the vehicle" %}</div>
+            <div>
+                <p>
+                    {% translate "THIS IS AUTOMATICALLY SEND MESSAGE, DO NOT RESPOND TO THIS" %} <br/>
+                    <br/>
+                    {% translate "Good partner!" %} <br/>
+                    <br/>
+                    {% translate "The low-emission vehicle parking benefit has been removed from the vehicle." %} <br/>
```

Comment:
Does text "The low-emission vehicle parking benefit has been removed from the vehicle." have a translation?

If so should be:

"The low-emission vehicle parking discount has been removed from the vehicle."